doc/dev/developer_guide/basic-workflow.rst

```diff
@@ -290,11 +315,16 @@ will need to force push your branch with:
 
    git push --force origin fix_1
 
+Why do we take these extra steps instead of simply adding additional commits
+the the PR?  It is best practice for a PR to consist of a single commit; this
+makes for clean history, eases peer review of your changes, and facilitates
+merges.  In rare circumstances it also makes it easier to cleanly revert
+changes.
+
 Merge
 -----
 
-The bugfixing process culminates when one of the project leads decides to
-merge your PR.
+The bugfix process completes when a project lead merges your PR.
```

Comment:
This is also good. I had been wanting to get rid of "culminates" for six months.

Reply:
It *was* a bit dramatic ;)